estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+import datetime as dt
+
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "An awesome estate module"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=dt.datetime.today() + dt.timedelta(days=90))
```

Comment:
```suggestion
    date_availability = fields.Date(copy=False, default=fields.Date.add(fields.Date.today(), months=3)
```
1. You have a `Date` type, therefore using `dt.datetime` instead of `dt.date` just add some extra useless casting.
2. What you did is mostly correct, just know that we have some Odoo built-ins (`fields.Date.add/subtract`) if you prefer.
3. Usually we take the biggest unit (`days` -> `months`).
4. Last small note: when `string` parameter is not provided to the field, Odoo will extrapolate a label to display in the view, in this case it will be `Date Availability` which is not super user friendly. But you adapted it in the view directly, so it works 👍 Just know that if someone create another view of the model, it will not be "Available From" but "Date Availability" by default.